content/millennium/r4/foundation/other/health-cards.md

```diff
@@ -43,11 +43,11 @@ Represents the patient and the clinical data related to a vaccination
 
 Contains the following resources:
 
-* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
   * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
   * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
 
-* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-immunization-dm.html) profile with the following fields, if valued:
+* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
```

Comment:
same here - think we need to link to the specific immunization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-vaccination-bundle-dm.html

Reply:
> same here - think we need to link to the specific immunization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-vaccination-dm.html

Fixed in commit https://github.com/cerner/fhir.cerner.com/pull/874/commits/6891a9e92a9faf1db82a825354e912180f12c6af